code/modules/projectiles/projectile/bullets.dm

```diff
@@ -53,6 +53,18 @@
 /obj/item/projectile/bullet/heavybullet
 	damage = 35
 
+/obj/item/projectile/bullet/sabot
+	damage = 150
+	var/breakthings = FALSE
+	armour_penetration = -5 //NOT A TYPO
+
+/obj/item/projectile/bullet/sabot/Range()
+	..()
+	damage += 70

```

Comment:
So it gains damage the further it goes? How many of these things do you intend to be shot?